examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/EventHubService.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Threading.Tasks;
+
+namespace WebDemoWithEventHub
+{
+    public class EventHubService
+    {
+        IConfigurationRoot _configRoot;
+
+        public EventHubService(IConfiguration config)
+        {
+            EventHubConnection eventHubConnection = config.GetSection("WebDemo:Connection").Get<EventHubConnection>();
+            InitEventHubProcessor(eventHubConnection);
+            if (config is IConfigurationRoot)
```

Comment:
What if this is false?

We should log error here. Use ILogger.

Reply:
Added a logger.

---
In reply to: [433597177](https://github.com/Azure/AppConfiguration/pull/329#discussion_r433597177) [](ancestors = 433597177)